### ISSUE TITLE
Spiderfoot won't start

### ISSUE THREAD

**Issue by capnspacehook** [smicallef/spiderfoot#126]
I have all the required dependencies installed, and still this happens. 

```user@kali-linux:~/spiderfoot$ ./sf.py 
Traceback (most recent call last):
  File "./sf.py", line 31, in <module>
    __import__(mod)
  File "/home/user/spiderfoot/ext/bs4/__init__.py", line 30, in <module>
    from .builder import builder_registry, ParserRejectedMarkup
  File "/home/user/spiderfoot/ext/bs4/builder/__init__.py", line 314, in <module>
    from . import _html5lib
  File "/home/user/spiderfoot/ext/bs4/builder/_html5lib.py", line 70, in <module>
    class TreeBuilderForHtml5lib(html5lib.treebuilders._base.TreeBuilder):
AttributeError: 'module' object has no attribute '_base'
```

**Comment by WebBreacher** [comment]
I am getting the same error on recently-patched Kali linux Python 2.7.13 with all required modules installed.

**Comment by smicallef** [comment]
I suspect there is a kind of conflict between the bs4 bundled with SpiderFoot and the one installed in Kali. It will be addressed in 2.9.

**Comment by smicallef** [comment]
Should now be addressed in 2.9.0.
